### PR TITLE
[mesheryctl] Pre-run check before checking system version

### LINKED ISSUE #2100: [mesheryctl] `version` fails to update when Docker is not running
**Description**
```
➜  ~ mesheryctl version
Client Version: v0.4.15 	  GitSHA: 51ca056186a824ad1c2b1bf979b95bdc46978d34
Server Version: unavailable 	  GitSHA: unavailable

Unable to communicate with Meshery: Get http://localhost:9081/api/server/version: dial tcp [::1]:9081: connect: connection refused
Ensure that Meshery is available, see Meshery Documentation (https://docs.meshery.io) for more help.

Checking for latest version of Meshery....

v0.4.15 is not the latest Meshery release. Update to v0.4.27. Run `mesheryctl system update`
✔ y
Updating Meshery now...
Traceback (most recent call last):
  File "site-packages/urllib3/connectionpool.py", line 677, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 392, in _make_request
  File "http/client.py", line 1252, in request
  File "http/client.py", line 1298, in _send_request
  File "http/client.py", line 1247, in endheaders
  File "http/client.py", line 1026, in _send_output
  File "http/client.py", line 966, in send
  File "site-packages/docker/transport/unixconn.py", line 43, in connect
ConnectionRefusedError: [Errno 61] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages/requests/adapters.py", line 449, in send
  File "site-packages/urllib3/connectionpool.py", line 727, in urlopen
  File "site-packages/urllib3/util/retry.py", line 403, in increment
  File "site-packages/urllib3/packages/six.py", line 734, in reraise
  File "site-packages/urllib3/connectionpool.py", line 677, in urlopen
  File "site-packages/urllib3/connectionpool.py", line 392, in _make_request
  File "http/client.py", line 1252, in request
  File "http/client.py", line 1298, in _send_request
  File "http/client.py", line 1247, in endheaders
  File "http/client.py", line 1026, in _send_output
  File "http/client.py", line 966, in send
  File "site-packages/docker/transport/unixconn.py", line 43, in connect
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionRefusedError(61, 'Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages/docker/api/client.py", line 205, in _retrieve_server_version
  File "site-packages/docker/api/daemon.py", line 181, in version
  File "site-packages/docker/utils/decorators.py", line 46, in inner
  File "site-packages/docker/api/client.py", line 228, in _get
  File "site-packages/requests/sessions.py", line 543, in get
  File "site-packages/requests/sessions.py", line 530, in request
  File "site-packages/requests/sessions.py", line 643, in send
  File "site-packages/requests/adapters.py", line 498, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionRefusedError(61, 'Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "docker-compose", line 3, in <module>
  File "compose/cli/main.py", line 67, in main
  File "compose/cli/main.py", line 123, in perform_command
  File "compose/cli/command.py", line 69, in project_from_options
  File "compose/cli/command.py", line 132, in get_project
  File "compose/cli/docker_client.py", line 43, in get_client
  File "compose/cli/docker_client.py", line 170, in docker_client
  File "site-packages/docker/api/client.py", line 188, in __init__
  File "site-packages/docker/api/client.py", line 213, in _retrieve_server_version
docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', ConnectionRefusedError(61, 'Connection refused'))
[8011] Failed to execute script docker-compose
Unable to update meshery: %wfailed to start meshery
See https://docs.meshery.io/guides/mesheryctl/#meshery-lifecycle-management for usage details
: exit status 255
```

**Expected Behavior**
`mesheryctl version` should invoke the same logic that `system update` does. When Docker is not running, the user should be prompted with:

```
Docker is not running.
Start Docker now [y/n]?
```

### PR DESCRIPTION
**Pull request by uds5501**
Signed-off-by: Uddeshya Singh <singhuddeshyaofficial@gmail.com>

**Description**
Runs additional pre-run check like `mesheryctl system update`

This PR fixes #2100

**Notes for Reviewers**
Sample logs are as follows

```cmd

Docker is not running.
Start Docker now [y/n]? y
Attempting to start Docker...
[sudo] password for uddeshya: 
Prerequisite Docker started.
```

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by kushthedude — APPROVED** [review]

**Comment by leecalcote** [comment]
@uds5501 thank you. We will want I land this check in the `update` flow. The title of error #2100 is accurate, but misleading. The error in #2100 occurred during an update operation. 

In addition to handling the update flow, another consideration here is to ensure that users are allowed to be able to check version without running Docker of Meshery server. In the case of them simply running “mesheryctl version”, we don’t want to prompt them to start Meshery server if it is not running. But, do want to prompt them to run Meshery if they try to update version.

**Comment by uds5501** [comment]
@leecalcote so basically this check should run once the user agrees for an update, correct?

**Review by leecalcote — APPROVED** [review]
Beautiful, @uds5501. Thanks for this. 👍

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 mesheryctl/cmd/mesheryctl/mesheryctl
 mesheryctl/.meshery/
 mesheryctl/mesheryctl
+mesheryctl/default.profraw
```
Comment (leecalcote) [inline .gitignore:5]: Thank you.

**Review by leecalcote — APPROVED** [review]
Thank you, @uds5501 😀

**Pull request merged**